network/src/commonMain/kotlin/com/wire/kalium/network/api/notification/EventContentDTO.kt

```diff
@@ -0,0 +1,46 @@
+package com.wire.kalium.network.api.notification
+
+import com.wire.kalium.network.api.ConversationId
+import com.wire.kalium.network.api.UserId
+import com.wire.kalium.network.api.notification.conversation.MessageEventData
+import com.wire.kalium.network.api.notification.user.NewClientEventData
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class EventResponse(
+    @SerialName("id") val id: String,
+    @SerialName("payload") val payload: List<EventContentDTO>?
```

Comment:
Maybe we can add `transient` field now to prevent refactoring later ?

Reply:
From transient docs:

> A boolean value. If true, the notification will only be received over a websocket, and not be stored on the backend for later retrieval. 

I wonder if it makes sense for us to ever handle it.

I'll add it anyway for now.

Reply:
> When receiving an event via websocket, the event can be marked as transient. A transient event will not be present in the notification stream, therefore it should not be stored as the last known notification.

https://wearezeta.atlassian.net/wiki/spaces/ENGINEERIN/pages/15566871/Incremental+synchronization#Storing-last-known-notification

If I understand well, transient events should not be pushed to notification stream

Reply:
@ohassine exactly.

So we will never receive them through `GET /notifications`. But we can receive through the WebSocket.

The thing is: If we are receiving them in real-time, then we process it aleady 😛.

We might add a storage for these events later and process them in the future. So I guess there's where the `transient` field comes in handy.